src/ngrest/plugins/Date.php

```diff
@@ -27,13 +27,21 @@ class Date extends Plugin
      */
     public $emptyMessage = '-';
     
+    /**
+     * @var string Use custom datetime format by [date filter](https://docs.angularjs.org/api/ng/filter/date). Default is 'shortDate'. Use false to take \yii\i18n\Formatter::$dateFormat as fallback.
+     * @since 2.0.0
+     */
+    public $format = 'shortDate';
+    
     /**
      * @inheritdoc
      */
     public function renderList($id, $ngModel)
     {
+        $format = $this->format ?? \Yii::$app->formatter->dateFormat;
```

Comment:
@boehsermoe please use `Yii` instead of `\Yii` we always add namespace to the namepsace declaration section. Sorry i just saw this now.